src/test/java/io/confluent/examples/streams/CustomStreamTableJoinIntegrationTest.java

```diff
@@ -280,8 +284,10 @@ public void shouldTriggerStreamTableJoinFromTable() throws Exception {
       //
       // Step 3: Verify the application's output data.
       //
-      final List<KeyValue<String, Long>> actualRecords = readOutputDataFromJoinedStream(expectedOutputRecords.size());
+      final List<KeyValue<String, Pair<Double, Long>>> actualRecords =
+          readOutputDataFromJoinedStream(expectedOutputRecords.size());
       assertThat(actualRecords).isEqualTo(expectedOutputRecords);
+      assertThat(actualRecords).containsExactlyElementsOf(expectedOutputRecords);
```

Comment:
nit: with this check do we need the `isEqualTo` check above still?

Reply:
Not needed, removed in follow-up commit.